p2p ut fail

--- FAIL: TestPrivateNetwork (1.26s)
    p2p_test.go:210: same privatenetwork test success
    p2p_test.go:226: 
        	Error Trace:	p2p_test.go:226
        	Error:      	Expected value not to be nil.
        	Test:       	TestPrivateNetwork
    p2p_test.go:228: different privatenetwork test success
    p2p_test.go:251: disable privatenetwork test success



https://github.com/33cn/chain33/pull/1087/checks?check_run_id=3320277661

--- a/system/p2p/dht/p2p_test.go
+++ b/system/p2p/dht/p2p_test.go
@@ -209,6 +209,7 @@ func TestPrivateNetwork(t *testing.T) {
 	assert.Nil(t, err)
 	t.Log("same privatenetwork test success")
 	h2.Close()
+	//断开与h2的网络连接
 	h1.Network().ClosePeer(h2.ID())
 	//h2 采用另外一种privatekey
 	var testPsk2 [32]byte

--- a/system/p2p/dht/p2p_test.go
+++ b/system/p2p/dht/p2p_test.go
@@ -209,6 +209,7 @@ func TestPrivateNetwork(t *testing.T) {
 	assert.Nil(t, err)
 	t.Log("same privatenetwork test success")
 	h2.Close()
+	h1.Network().ClosePeer(h2.ID())
 	//h2 采用另外一种privatekey
 	var testPsk2 [32]byte
 	copy(testPsk2[:], testPSK)

